### PR TITLE
update gitignore and remove all editor settings from the repo

### PR DESCRIPTION
**Pull request by woutdenolf**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
-manual/makelog.txt
-/build/
+
+# Unknown
 /python/
-impatient-guide/_build
-.DS_Store
 __github_creds__.txt
-
-# Microsoft VSCode editor configuration
```
Comment (prjemian) [inline .gitignore:20]: This is still a good protection to keep.  It prevents unwanted content from creeping in by accident.

**Review by prjemian — COMMENTED** [review]
I like it.  One detail to keep, as noted.

File: .gitignore (hunk 1 above)
Comment (woutdenolf) [reply .gitignore:20]: That's taken care of by the more general

```
# Hidden files
.*
```

Should stop all editors like eclipse, vscode, pycharm, ...

**Comment by PeterC-DLS** [comment]
Just wondering why you want to be rid of IDE project files?

**Review by prjemian — APPROVED** [review]
OK

**Pull request merged**